NEWS

```diff
@@ -1,6 +1,75 @@
 This file describes changes in recent versions of Slurm. It primarily
 documents those changes that are of interest to users and administrators.
 
+* Changes in Slurm 19.05.7
```

Comment:
@itkovian you update to `Update to 19.05.6` but this has changes from `Update to 19.05.7` ? I'm confused.

Reply:
There is a NEWS section for `.07` but it is not yet finished or at that version. Confuses me too.